conf/tentacle/rados/test-confs/pool-configurations.yaml

```diff
@@ -59,3 +59,32 @@ erasure:
     k: 2
     m: 2
     crush-failure-domain: host
+  sample-pool-7:
```

Comment:
You're adding the same content in both the PR's. It's gonna cause merge conflict on the other PR.